nativelib/src/main/resources/platform.c

```diff
@@ -0,0 +1,20 @@
+#ifdef _WIN32
+#include <Windows.h>
+#endif
+
+int scalanative_platform_is_windows() {
```

Comment:
@jonas We'll probably go with more comprehensible design in the future, right now it's not meant to be a finished public API yet (that's why it's in runtime namespace.)